coderd/database/queries/siteconfig.sql

```diff
@@ -43,6 +43,13 @@ ON CONFLICT (key) DO UPDATE SET value = $1 WHERE site_configs.key = 'service_ban
 -- name: GetServiceBanner :one
 SELECT value FROM site_configs WHERE key = 'service_banner';
 
+-- name: UpsertNotificationBanners :exec
+INSERT INTO site_configs (key, value) VALUES ('notification_banners', $1)
+ON CONFLICT (key) DO UPDATE SET value = $1 WHERE site_configs.key = 'notification_banners';
+
+-- name: GetNotificationBanners :one
+SELECT value FROM site_configs WHERE key = 'notification_banners';
```

Comment:
I wonder if we should name the key like `notification_banners_json` to signal this is a json payload. I know `service_banner` is named without it. We can write a migration to remove `service_banner` entirely from the db.

Reply:
There's a migration to convert any existing `'service_banner'` row into a `'notification_banners'` row. :) and I think the code makes it pretty clear that the row contains JSON